File: Personal/ComicAndPicture/ComicAndPicture/Models/CategoryModel.m
//
//  CategoryModel.m
//  ComicAndPicture
//
//  Created by MS on 16/3/12.
//  Copyright © 2016年 JFY. All rights reserved.
//

#import "CategoryModel.h"

@implementation CategoryModel

- (void)setValue:(id)value forUndefinedKey:(NSString *)key {
    
    if ([key isEqualToString:@"id"]) {
        
        self.categoryId = value;
        
    }
    
}

@end

File: Personal/小项目/ComicAndPicture/ComicAndPicture/Model/CategoryModel.swift
//
//  CategoryModel.swift
//  ComicAndPicture
//
//  Created by vcyber on 16/11/7.
//  Copyright © 2016年 vcyber. All rights reserved.
//

import UIKit

class CategoryModel: NSObject {

    var categoryId:String = ""
    var name:String = ""
    var images:String = ""
    
    override func setValue(_ value: Any?, forUndefinedKey key: String) {
        if key == "id" {
            self.categoryId = value as! String
        }
    }
    
    init(dict:Dictionary<String, Any>){
        super.init()
        setValuesForKeys(dict)
    }
}
